ict-weather-station/transformers/main/source.js

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * 
+ *  Transformation code for ICT Weather Station Sensor 
+ * 
+ * ****************************************************************************/
+
+/**
+ * Extracts bytes from byte_payload from start to end and converts these bytes 
+ * into a float. Adapted from: 
+ * https://www.thethingsnetwork.org/forum/t/decode-float-sent-by-lopy-as-node/8757/2
+ * 
+ * @param {*} byte_payload payload in bytes to be converted to float.
+ * @param {*} start starting byte index in payload.
+ * @param {*} end ending byte index in payload.
+ */
+function extractFloat(byte_payload, start, end) {
```

Comment:
Perhaps `decodeFloat` may be a better name? Up to you.

Reply:
Function name is now decodeFloat.